src/screens/HyperSwitch/Utils/CurrencyUtils.res

```diff
@@ -310,358 +310,3 @@ let getCurrencyCodeStringFromVariant = currencyCode => {
   | USD => "USD"
   }
 }
-
-let getCurrencyCodeFromString: string => currencyCode = currencyCode => {
```

Comment:
Can you please revert this file as it might be needed.

Reply:
Done